1-JavaScript/Week3/assignment/ex2-mondaysWorth.test.js

```diff
@@ -30,10 +30,15 @@ const mondayTasks = [
 
 const hourlyRate = 25;
 
-function computeEarnings(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function computeEarnings(mondayTasks, hourlyRate) {
+  const results = mondayTasks.map(task => task.duration/60 * hourlyRate)
+  const sumHours = results.reduce((sum, current) => sum + current, 0)
```

Comment:
This is a nice structure, In this case, you can combine the functions cause you only want the result.
```suggestion
   const results = mondayTasks
    .map((task) => (task.duration / 60) * hourlyRate)
    .reduce((sum, current) => sum + current, 0);
```